dashboard/db/migrate/20171013180932_backfill_regional_partner_program_managers_from_contacts.rb

```diff
@@ -0,0 +1,16 @@
+class BackfillRegionalPartnerProgramManagersFromContacts < ActiveRecord::Migration[5.0]
+  def up
+    RegionalPartner.joins(:contact).each do |regional_partner|
+      begin
+        regional_partner.program_manager = regional_partner.contact.id
+      rescue RuntimeError => e
+        # Log the error but don't fail the migration if something goes wrong.
+        CDO.logger.error "Error converting Regional Partner (#{regional_partner.id})  "\
+          "Contact (#{regional_partner.contact.id}) into a Program Manager: #{e.message}"
```

Comment:
Why? Are we expecting errors? If so, better to handle them explicitly. Otherwise I'd rather it blow up if it fails, we'll notice and respond.